### PR TITLE
45 unify scaladocs

### PR DESCRIPTION
**Pull request by flsh86**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+[![Join the chat at https://gitter.im/softwaremill/sttp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/softwaremill/sttp)
+[![CI](https://github.com/softwaremill/sttp-openai/workflows/CI/badge.svg)](https://github.com/softwaremill/sttp-openai/actions?query=workflow%3ACI+branch%3Amaster)
+
+[//]: # ([![Maven Central]&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;)
+
+sttp is a family of Scala HTTP-related projects, and currently includes:
+
+* [sttp client](https://github.com/softwaremill/sttp): The Scala HTTP client you always wanted!
+* [sttp tapir](https://github.com/softwaremill/tapir): Typed API descRiptions
+* sttp openai: this project. Scala client wrapper for OpenAi API. Use the power of ChatGPT inside your code!
```
Comment (kciesielski) [inline README.md:12]: **suggestion:** Use `OpenAI` in such context, we wanted `OpenAi` only as a convention for Scala type names.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+![sttp-model](https://github.com/softwaremill/sttp-openai/raw/master/banner.png) Banner should be changed to sttp openAi
```
[the remark this answers is not in the document]
Comment (flsh86) [reply README.md:1]: I left it for myself, before we had new banner and forgot to delete it 
Thanks for catching it

File: core/src/main/scala/sttp/openai/requests/completions/chat/Message.scala [hunk 3]
```diff
@@ -2,6 +2,11 @@ package sttp.openai.requests.completions.chat
 
 import sttp.openai.json.SnakePickle
 
+/** @param role
```
[the remark this answers is not in the document]
Comment (flsh86) [reply core/src/main/scala/sttp/openai/requests/completions/chat/Message.scala:5]: I've created an issue for that, thanks for pointing it out :)

**Comment by adamw** [comment]
What about comments on the request description methods? (w/ links to open ai reference docs)

**Comment by benzwreck** [comment]
> What about comments on the request description methods? (w/ links to open ai reference docs)

@adamw What do you exactly mean? `OpenAi.scala` seems to have them all.

**Comment by adamw** [comment]
@benzwreck sorry, I retracted this on the other PR, it's just that GH folded the file so I skipped it a first ;) There's no problem :)

**Review by kciesielski — APPROVED** [review]

**Pull request merged**